### ISSUE TITLE
Crash on "Could not find object with given id"

### ISSUE THREAD

**Issue by jonahwilliams** [dart-lang/webdev#727]
This one seems like a case that should be handled in dwds.

Use Command: `run --machine --target lib/main.dart -d chrome --track-widget-creation --start-paused`

```
Thread 0 main thread CRASHEDWipError: WipError 16: {code: -32000, message: Could not find object with given id}
at Debugger.getProperties	(debugger.dart:368 )
at <asynchronous gap>	(async )
at visibleProperties.<anonymous closure>	(dart_scope.dart:29 )
at <asynchronous gap>	(async )
at MappedListIterable.elementAt	(iterable.dart:415 )
at ListIterable.toList	(iterable.dart:219 )
at visibleProperties	(dart_scope.dart:30 )
at _AsyncAwaitCompleter.start	(async_patch.dart:43 )
at visibleProperties	(dart_scope.dart:18 )
at Debugger.variablesFor	(debugger.dart:335 )
at _AsyncAwaitCompleter.start	(async_patch.dart:43 )
at Debugger.variablesFor	(debugger.dart:332 )
at Debugger.dartFramesFor	(debugger.dart:322 )
at _asyncThenWrapperHelper.<anonymous closure>	(async_patch.dart:71 )
at _rootRunUnary	(zone.dart:1132 )
at _CustomZone.runUnary	(zone.dart:1029 )
at _FutureListener.handleValue	(future_impl.dart:137 )
at Future._propagateToListeners.handleValueCallback	(future_impl.dart:678 )
at Future._propagateToListeners	(future_impl.dart:707 )
at Future._completeWithValue	(future_impl.dart:522 )
at _AsyncAwaitCompleter.complete	(async_patch.dart:30 )
at _completeOnAsyncReturn	(async_patch.dart:288 )
at Debugger._frameFor	(debugger.dart )
at _asyncThenWrapperHelper.<anonymous closure>	(async_patch.dart:71 )
at _rootRunUnary	(zone.dart:1132 )
at _CustomZone.runUnary	(zone.dart:1029 )
at _FutureListener.handleValue	(future_impl.dart:137 )
at Future._propagateToListeners.handleValueCallback	(future_impl.dart:678 )
at Future._propagateToListeners	(future_impl.dart:707 )
at Future._completeWithValue	(future_impl.dart:522 )
at Future._asyncComplete.<anonymous closure>	(future_impl.dart:552 )
at _rootRun	(zone.dart:1124 )
at _CustomZone.run	(zone.dart:1021 )
at _CustomZone.runGuarded	(zone.dart:923 )
at _CustomZone.bindCallbackGuarded.<anonymous closure>	(zone.dart:963 )
at _microtaskLoop	(schedule_microtask.dart:41 )
at _startMicrotaskLoop	(schedule_microtask.dart:50 )
at _runPendingImmediateCallback	(isolate_patch.dart:116 )
at _RawReceivePortImpl._handleMessage	(isolate_patch.dart:173 )
```

**Comment by jakemac53** [comment]
cc @alan-knight any thoughts on what to do here?

We could just log a warning in the console and continue on. It looks like possibly we are trying to load the variables in scope but there was a race condition and they went out of scope before we finished. Maybe the user hitting continue back to back quickly could cause this, or maybe our built in `step out` functionality for the sdk could cause it?

**Comment by alan-knight** [comment]
I don't think our step out could cause it, but a fast pause/resume might. I don't see anything sensible to do there but suppress it and return an empty variable list.

**Comment by jakemac53** [comment]
SGTM

**Comment by alan-knight** [comment]
I think this is fixed by #732 
Now it will catch errors in our Chrome event handlers and log them, but not stop.
